docs/tutorials/standalone/Bucket/bucket_tutorial.jl

```diff
@@ -327,14 +327,18 @@ prob = SciMLBase.ODEProblem(
 );
 
 # We need a callback to get and store the auxiliary fields, as they
-# are not stored by default.
+# are not stored by default. We also need a callback to update the
+# drivers (atmos and radiation)
 saveat = collect(t0:Δt:tf);
 saved_values = (;
     t = Array{Float64}(undef, length(saveat)),
     saveval = Array{NamedTuple}(undef, length(saveat)),
 );
-
-cb = ClimaLSM.NonInterpSavingCallback(saved_values, saveat);
+saving_cb = ClimaLSM.NonInterpSavingCallback(saved_values, saveat);
+updateat = deepcopy(saveat)
```

Comment:
I think we want `copy`, not `deepcopy` in this particular case.

```julia
julia> a = [1, 2, 3]
3-element Vector{Int64}:
 1
 2
 3

julia> b = copy(a)
3-element Vector{Int64}:
 1
 2
 3

julia> b[2] = 10
10

julia> b
3-element Vector{Int64}:
  1
 10
  3

julia> a
3-element Vector{Int64}:
 1
 2
 3
```

Reply:
thanks! i changed this.